### ISSUE TITLE
Typo in CMakeLists.txt comment

### ISSUE THREAD

**Issue by duncanspumpkin** [DanielChappuis/reactphysics3d#315]
https://github.com/DanielChappuis/reactphysics3d/blob/17dd22e677ed861b0d4ece0c00a7e3cb503cc2f0/CMakeLists.txt#L292
Think this was copied from https://github.com/pabloariasal/modern-cmake-sample/blob/master/libjsonutils/CMakeLists.txt without first adjusting the comment. 

Comment should be modified from:

```
# This is required so that the exported target has the name JSONUtils and not jsonutils
```
To:
```
# This is required so that the exported target has the name ReactPhysics3D and not reactphysics3d
```

**Comment by DanielChappuis** [comment]
Yes you are right. Thanks a lot for taking the time to report this. I will fix this in the next release version of the library.

**Comment by DanielChappuis** [comment]
This is now fixed in version [v0.10.0](https://github.com/DanielChappuis/reactphysics3d/releases/tag/v0.10.0) of the library. Thanks a lot for reporting the issue.
